packages/shorebird_cli/lib/src/validators/android_internet_permission_validator.dart

```diff
@@ -90,4 +91,20 @@ class AndroidInternetPermissionValidator extends Validator {
     return attribute.qualifiedName == 'android:name' &&
         attribute.value == 'android.permission.INTERNET';
   }
+
+  void _addInternetPermissionToFile(String path) {
+    final xmlDocument = XmlDocument.parse(File(path).readAsStringSync());
```

Comment:
The user would probably see an ugly error. I'm not particularly worried about this throwing because it's private to this class and the we only provide it the results of `listSync`.